lib/config/options/index.ts

```diff
@@ -1378,7 +1378,7 @@ const options: RenovateOptions[] = [
     description: 'Use the default reviewers (Bitbucket only).',
     type: 'boolean',
     default: true,
-    supportedPlatforms: ['bitbucket'],
+    supportedPlatforms: ['bitbucket', 'bitbucket-server'],
```

Comment:
I'm not sure this is right, as we have totally different implementations for those platforms.

Reply:
Thats correct, but config option is used in both:
**bitbucket:**
https://github.com/renovatebot/renovate/blob/711dde088b99de8c2af55c2b878d4767b1ee6450/lib/platform/bitbucket/index.ts#L677

**bitbucket-server**
https://github.com/renovatebot/renovate/blob/2ec5c54cf0612c192a426afb4f2cba0e2e9441f2/lib/platform/bitbucket-server/index.ts#L813